### ISSUE TITLE
Replace  Session.getDefaultInstance(...) with Session.getInstance(...)

### ISSUE THREAD

**Issue by open-gdsn** [difi/oxalis#445]
The `Session.getDefaultInstance(...)` should rarely be used according to

 -  https://javaee.github.io/javamail/FAQ#commonmistakes
- https://stackoverflow.com/questions/4184204/what-is-the-difference-between-getdefaultinstance-and-getinstance-in-session
 - https://stackoverflow.com/questions/22657918/what-is-the-use-of-javax-mail-session/22664494

Is there a reason Oxalis does this? It makes it hard to integrate with other applications/frameworks/... If there is no specific reason, can it be replaced?

**Comment by kukel** [comment]
Yes I want this too... Remarkable there is no feedback on this issue from Oxalis
